.travis.yml

```diff
@@ -1,102 +1,118 @@
-  - echo "sendmail_path='true'" >> `php --ini | grep "Loaded Configuration" | awk '{print $4}'`
-  # Forward the errors to the syslog so we can print them
-  - echo "error_log=syslog" >> `php --ini | grep "Loaded Configuration" | awk '{print $4}'`
-  # Get latest drupal 8 core
-  - cd $TRAVIS_BUILD_DIR/..
-  - git clone --depth 1 --branch 8.0.x http://git.drupal.org/project/drupal.git
-  # Restart apache and test it
-  - sudo service apache2 restart
-  - curl -v "http://localhost"
-  # Re-enable when trying to get CodeSniffer doesn't return a 403 anymore.
-  #- composer global require drupal/coder:\>7
+  - drupal-ti install
 
 before_script:
-  - cd $TRAVIS_BUILD_DIR/../drupal
-  # Mysql might time out for long tests, increase the wait timeout.
-  - mysql -e 'SET @@GLOBAL.wait_timeout=1200'
-  # Update drupal core
-  - git pull origin 8.0.x
-  # Install the site
-  - drush -v site-install minimal --db-url=mysql://root:@localhost/drupal --yes
-  - drush en --yes simpletest
-  - drush cr
-  - phpenv rehash
+  - drupal-ti before_script
 
 script:
-  # go to our Drupal module directory
-  - mkdir $TRAVIS_BUILD_DIR/../drupal/modules/redirect
-  - cp -R $TRAVIS_BUILD_DIR/* $TRAVIS_BUILD_DIR/../drupal/modules/redirect/
-  # go to our Drupal main directory
-  - cd $TRAVIS_BUILD_DIR/../drupal
-  - ls -la $TRAVIS_BUILD_DIR/../drupal/sites/default
-  # Run the tests
-  - php core/scripts/run-tests.sh --verbose --color --concurrency 4 --php `which php` --url $URL "redirect" | tee /tmp/test.txt; TEST_EXIT=${PIPESTATUS[0]}; echo $TEST_EXIT
-  # Check if we had fails in the run-tests.sh script
-  # Exit with the inverted value, because if there are no fails found, it will exit with 1 and for us that\
-  # is a good thing so invert it to 0. Travis has some issues with the exclamation mark in front so we have to fiddle a
-  # bit.
-  # Also make the grep case insensitive and fail on run-tests.sh regular fails as well on fatal errors.
-  - TEST_OUTPUT=$(! egrep -i "([0-9]+ fails)|(PHP Fatal error)|([0-9]+ exceptions)" /tmp/test.txt > /dev/null)$?
-  - echo $TEST_OUTPUT
-  - cd $TRAVIS_BUILD_DIR/../drupal/core
-  - ./vendor/bin/phpunit --verbose --debug ../modules/redirect/; TEST_PHPUNIT=$?; echo $TEST_PHPUNIT
-  # if the TEST_EXIT status is 0 AND the TEST_OUTPUT status is also 0 it means we succeeded, in all other cases we
-  # failed.
-  # Re-enable when trying to get CodeSniffer doesn't return a 403 anymore.
-  #- /home/travis/.composer/vendor/bin/phpcs --standard=/home/travis/.composer/vendor/drupal/coder/coder_sniffer/Drupal --extensions=php,inc,test,module,install --ignore=css/ $TRAVIS_BUILD_DIR/../drupal/modules/search_api
-  - php -i | grep 'php.ini'
-  - sudo cat /var/log/apache2/error.log
-  # Exit the build
-  - if [ $TEST_EXIT -eq 0 ] && [ $TEST_OUTPUT -eq 0 ] && [ $TEST_PHPUNIT -eq 0 ]; then exit 0; else exit 1; fi
+  - drupal-ti script
+
+after_script:
+  - drupal-ti --include .travis-after-script.sh
```

Comment:
Uhm, this was meant to be called like:

```
.travis-after-before_script.sh
```

And then be called in the before_script section after the before_script command.

Likely could also call it in the script section before the ```drupal-ti script``` though.

which probably is best.